parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -34,7 +34,7 @@ installDeps() {
     local OSVERSION
     OSVERSION=$(grep DISTRIB_RELEASE /etc/*-release| cut -f 2 -d "=")
     BLOBFUSE_VERSION="1.4.5"
-    BLOBFUSE2_VERSION="2.1.2"
+    BLOBFUSE2_VERSION="2.2.0"
```

Comment:
Did you test this on AKS with this change?

Reply:
yes, I have tested this change in CSI driver repo on Ubuntu 22.04: https://github.com/kubernetes-sigs/blob-csi-driver/pull/1235